lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/client/LoggingJobStatusHandler.java

```diff
@@ -28,7 +28,9 @@
 /**
  * A handler for job status messages that just logs them.
  */
-final class LoggingJobStatusHandler implements EventHandler<ReefServiceProtos.JobStatusProto> {
+@DriverSide
+final class LoggingJobStatusHandler implements JobStatusHandler {
```

Comment:
I agree. I've created a JIRA [REEF-1596](https://issues.apache.org/jira/browse/REEF-1596) and will push the changes in a separate PR later.